docs/operator-manual/rbac.md

```diff
@@ -155,7 +155,27 @@ p, example-user, applications, delete/*/Pod/*/*, default/prod-app, allow
 
     ```csv
     p, example-user, applications, delete, default/prod-app, allow
-    p, example-user, applications, delete/*/Pod/*/*, default/prod-app, deny
+    p, example-user, applications, delete/*/Pod/*, default/prod-app, deny
+    ```
+
+!!! note
+
+    In v3, RBAC will have a breaking change. The `update` and `delete` actions
+    (without a `/*`) will no longer include sub-resources. This allows you to
+    explicitly allow or deny access to an application without affecting its
+    sub-resources. For example, you may want to allow enable/disable of auto-sync
```

Comment:
> you may want to allow enable/disable of auto-sync
    by allowing update on an application

Personally, I wouldn't give the update permission for this usecase. The update permissions are still too wide and allow you to fully change the source/destination/project and many more.

If you just want the users to be able to enable/disable auto-sync, I would suggest you to create a resource actions `toggle-auto-sync` on the Application resource, and control the permissions with the action rbac.